pkgs/development/python-modules/playwright/default.nix

```diff
@@ -63,9 +64,13 @@ buildPythonPackage rec {
     substituteInPlace setup.py \
       --replace "self._download_and_extract_local_driver(base_wheel_bundles)" ""
 
+    # Set the correct nodejs path
+    substituteInPlace playwright/_impl/_driver.py \
+      --replace "@nodejs@" "${nodejs}/bin/node"
```

Comment:
dont you get warnings there ? should be a variant like `--replace-fail`

Reply:
no warnings, it works with `--replace-fail` as well